src/assets/scripts/base/PositionModel.js

```diff
@@ -267,49 +247,26 @@ export default class PositionModel {
  */
 PositionModel.calculatePosition = (coordinates, referencePostion, magneticNorth) => {
     if (!coordinates || !referencePostion || !magneticNorth) {
-        throw new TypeError('Invalid parameter. PositionModel.getPosition() requires coordinates, referencePostion and magneticNorth as parameters');
-    }
-
-    let latitude;
-    let longitude;
-    let rawX = 0;
-    let rawY = 0;
-
-    if (!hasCardinalDirectionInCoordinate(coordinates[LATITUDE_INDEX])) {
-        rawX = coordinates[LATITUDE_INDEX];
-        rawY = coordinates[LONGITUDE_INDEX];
+        throw new TypeError(`Invalid parameter. PositionModel.getPosition() requires
+                            coordinates, referencePostion and magneticNorth as parameters`);
```

Comment:
Asked about this in slack as well... is this an acceptable resolution to the lint warning due to length of the line?